less/paperhive/login.less

```diff
@@ -0,0 +1,82 @@
+.ph-login-icon {
+  width: 18px;
+  height: 18px;
+  vertical-align: baseline;
+}
+
+.ph-login-separator {
+  color: #E0E0E0;
+}
+
+.ph-login-line-separator {
+  width: 1px;
+  background-color: #E0E0E0;
+  height: 12em;
+}
+
+.ph-flex-container {
+  display: -webkit-flex;
+  display: flex;
+  -webkit-justify-content: center;
+  justify-content: center;
+}
+
+.ph-flex-item {
+  -webkit-align-self: center;
+  align-self: center;
+  -webkit-flex: 3;
+  flex: 3;
+}
+
+.ph-flex-item-1 {
+  -webkit-align-self: center;
+  align-self: center;
+  -webkit-flex: 1;
+  flex: 1;
+}
+
+.ph-inner-flex-container {
+  display: -webkit-flex;
+  display: flex;
+  -webkit-flex-direction: column;
+  flex-direction: column;
+}
+
+.ph-inner-flex-item {
+  -webkit-align-self: center;
+  align-self: center;
+  -webkit-flex: 1;
+  flex: 1;
```

Comment:
Oh, I didn't check firefox...

`flex` defines the length of the flex item relative to the other items inside the `flex-container`. So I thought we need this. But if it also works without this CSS code in chrome, please remove it.